src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmReplyService.java

```diff
@@ -287,7 +285,8 @@ private boolean isResponseRequired(CaseData caseData, String title) {
     private Map<String, String> buildPersonalisation(String caseNumber, String caseId, String customText) {
         Map<String, String> personalisation = new ConcurrentHashMap<>();
         personalisation.put(CASE_NUMBER, caseNumber);
-        personalisation.put(CASE_ID, caseId);
+        personalisation.put(LINK_TO_CITIZEN_HUB, notificationProperties.getCitizenUrl() + caseId);
+        personalisation.put(LINK_TO_EXUI, notificationProperties.getExuiUrl() + caseId);
```

Comment:
Perhaps we can create a method inside notificationProperties that stitches these two parts together, theres a lot of repeated code of "URL + caseId". Calling a method that returns this might be a bit cleaner/simpler